example/lib/invenio-search-js.js

```diff
@@ -0,0 +1,303 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2015 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+* In applying this license, CERN does not
+* waive the privileges and immunities granted to it by virtue of its status
+* as an Intergovernmental Organization or submit itself to any jurisdiction.
+*/
+
+(function (angular) {
+
+  /**
+    * @namespace InvenioSearch
+    * @desc Create and make the request
+    * @memberOf Controllers
+    */
+  function invenioSearchController(invenioSearchAPI) {
+
+    this.invenioDoSearch = invenioDoSearch;
+    this.invenioResults = [];
+    this.invenioSearchError = false;
+    this.invenioSearchLoading = false;
+    this.invenioSearchQuery = '';
+
+    // We don't have any logic on the controller until search initialized
+    this.invenioSearchArgs = {
+      url: '',
+      method: 'GET',
+      params: {
+        page: 1,
+        size: 10,
+        q: '',
+        index: '',
+        doc_type: '',
+      }
+    };
+
+    ////////////
+
+    /**
+    * @name invenioDoSearch
```

Comment:
@drjova can you check the indentation here (see https://github.com/inveniosoftware/invenio-search-js/pull/1/files#diff-fc0759b7396267f05828baca8f6f544cR26)